client/jetpack-connect/jetpack-connect-notices.jsx

```diff
@@ -335,10 +336,13 @@ export class JetpackConnectNotices extends Component {
 		return null;
 	}
 }
-export default connect( state => {
-	const jetpackConnectSite = getConnectingSite( state );
-	const siteToConnect = jetpackConnectSite.url;
-	return {
-		siteToConnect,
-	};
-} )( localize( JetpackConnectNotices ) );
+export default connect(
+	state => {
+		const jetpackConnectSite = getConnectingSite( state );
+		const siteToConnect = jetpackConnectSite.url;
```

Comment:
Just to be on the safe side, would you like to use a `get()` to retrieve the `url` here?